pom.xml

```diff
@@ -235,6 +235,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>animal-sniffer-maven-plugin</artifactId>
+                <version>1.14</version>
+                <configuration>
+                    <signature>
+                        <groupId>org.codehaus.mojo.signature</groupId>
+                        <artifactId>java16-sun</artifactId>
+                        <version>1.0</version>
```

Comment:
Latest version might be [1.1](http://search.maven.org/#browse%7C-379312183). Or is there a reason you used version 1.0 here?